hack/lib/build/version.sh

```diff
@@ -40,10 +40,11 @@ function os::build::version::openshift_vars() {
 			# Try to match the "git describe" output to a regex to try to extract
 			# the "major" and "minor" versions and whether this is the exact tagged
 			# version or whether the tree is between two tagged versions.
-			if [[ "${OS_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)(\.[0-9]+)*([-].*)?$ ]]; then
+			if [[ "${OS_GIT_VERSION}" =~ ^v([0-9]+)\.([0-9]+)\.([0-9]+)(\.[0-9]+)*([-].*)?$ ]]; then
 				OS_GIT_MAJOR=${BASH_REMATCH[1]}
 				OS_GIT_MINOR=${BASH_REMATCH[2]}
-				if [[ -n "${BASH_REMATCH[4]}" ]]; then
+				OS_GIT_PATCH=${BASH_REMATCH[3]}
+				if [[ -n "${BASH_REMATCH[5]}" ]]; then
```

Comment:
You mean if matches the last group, `([-].*)?` right? Isn't this in `BASH_REMATCH[5]` now, since I added a new regex group, and the `+="+"` is in the line below? Not supposed to have changed.